### PR TITLE
initiate quick start feature guide and add simple pod scheduling

### PR DESCRIPTION
**Pull request by yuey002**
* Initiate the structure for quick start feature guide
* Modify the initial quick-start to a simple pod scheduling guide

### CODE REVIEW

File: manifests/base/crds/kustomization.yaml
Comment (lintongj) [inline manifests/base/crds/kustomization.yaml:1]: Do we have to update the file permissions for this file and ones below?

File: docs/features/kind-cluster-setup.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Quickstart
-This Quickstart guide will walk you through how to set up the Gödel Unified Scheduling system.
+# Local Gödel Environment Set-Up with KIND
```
Comment (lintongj) [inline docs/features/kind-cluster-setup.md:1]: Set-Up --> Setup

File: README.md [hunk 2]
```diff
@@ -13,8 +13,14 @@ Thanks to the Gödel architecture, we can realize enhanced performance in terms
 
 - While remaining compatible with the Kubernetes ecosystem and serving as a potential substitute for the Kubernetes scheduler, our framework distinguishes itself through performance and architectural advancements. Although the framework interface deviates slightly from that of the Kubernetes scheduler, its extensibility remains uncompromised. Scheduling plugins can be developed in a manner akin to Kubernetes.
 
-## Quick Start Guide
-Gödel supports Kubernetes versions from 1.18 up to 1.24.6. Using lower or higher Kubernetes versions may cause compatibility issues. For setup please refer to [Quickstart](./docs/quickstart.md).
+## Quick Start Guide for Selected Features
```
Comment (lintongj) [inline README.md:16]: Key Features

File: manifests/base/crds/kustomization.yaml
Comment (yuey002) [reply manifests/base/crds/kustomization.yaml:1]: Nope, reverted

File: docs/features/kind-cluster-setup.md (hunk 1 above)
Comment (yuey002) [reply docs/features/kind-cluster-setup.md]: Fixed

File: README.md (hunk 2 above)
Comment (yuey002) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -13,8 +13,14 @@ Thanks to the Gödel architecture, we can realize enhanced performance in terms
 
 - While remaining compatible with the Kubernetes ecosystem and serving as a potential substitute for the Kubernetes scheduler, our framework distinguishes itself through performance and architectural advancements. Although the framework interface deviates slightly from that of the Kubernetes scheduler, its extensibility remains uncompromised. Scheduling plugins can be developed in a manner akin to Kubernetes.
 
-## Quick Start Guide
-Gödel supports Kubernetes versions from 1.18 up to 1.24.6. Using lower or higher Kubernetes versions may cause compatibility issues. For setup please refer to [Quickstart](./docs/quickstart.md).
+## Quick Start Guide for Key Features
+Please refer to the below links for quick start guides on key features. Please note that Gödel supports Kubernetes versions from 1.18 up to 1.24.6. Using lower or higher Kubernetes versions may cause compatibility issues.
+
+- [Basic Pod Scheduling](./docs/features/basic-pod.md)
+- Gang Scheduling
```
Comment (NickrenREN) [inline README.md:20]: when features quickstart docs are ready, we need to go back to this doc and link to those docs ,right ?

**Comment by NickrenREN** [comment]
Also, please sign the CLA

File: README.md (hunk 3 above)
Comment (yuey002) [reply README.md:20]: Yes that's the plan. We'll create dedicated pages and link from here.

**Comment by yuey002** [comment]
> Also, please sign the CLA

@NickrenREN Done

File: README.md (hunk 3 above)
Comment (NickrenREN) [reply README.md:20]: get it, so can we add a TODO comment here frist

**Comment by NickrenREN** [comment]
> > Also, please sign the CLA
> 
> @NickrenREN Done

seems the CLA check is still red

**Comment by yuey002** [comment]
> > > Also, please sign the CLA
> > 
> > 
> > @NickrenREN Done
> 
> seems the CLA check is still red

This is weird.. I have signed CLA multiple times and can see 'CLA signed' in the history.
image

File: README.md (hunk 3 above)
Comment (yuey002) [reply README.md:21]: Added the TODO comment

**Comment by yuey002** [comment]
> > > Also, please sign the CLA
> > 
> > 
> > @NickrenREN Done
> 
> seems the CLA check is still red

It's passed now. Turns out the username & email was wrong yesterday..

File: docs/features/basic-pod.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Quickstart - Basic Pod Scheduling
+This Quickstart guide will walk you through how to schedule a pod using the Gödel Scheduling System.
+
+## Local Cluster Bootstrap & Installation
+
+Please refer to [Cluster Set-Up Guide](./docs/features/kind-cluster-setup.md) for creating a local KIND cluster installed with Gödel.
```
Comment (lintongj) [inline docs/features/basic-pod.md:6]: 1. Set-Up -> Setup
2. the link is broken as the relative path is not correct

Comment (yuey002) [reply docs/features/basic-pod.md]: Fixed both

**Comment by yuey002** [comment]
@NickrenREN Could you review and see if you have other comments? Thank you.

**Review by NickrenREN — APPROVED** [review]

**Pull request merged**